### ISSUE TITLE
Forum

### ISSUE THREAD

**Issue by curtisdelicata** [familytree365/nuxt#40]
This is  a **bug | feature request**.

### Prerequisites
* [ ] Are you running the latest version?
* [ ] Are you reporting to the correct repository?
(enso is made of many specialized packages: https://github.com/familytree365)
* [ ] Did you check the documentation?
* [ ] Did you perform a cursory search?

**Comment by curtisdelicata** [comment]
Standard features, let admin roles create categories and anyone can create topics. Maybe make use of the Enso components for discussions or comments.

**Comment by windrunnerdev** [comment]
@everyone you are scammed, please check @curtisdelicata 's Linkedin profile. He is fraud and liar. He never pay for developers. 

He runs fake business called FamilyTree365. He lies developers to contribute the project for a week and pretend to pay after a week but he disappears after receiving PR.

I should alert other honest developers and customers that you should never support Curtis Delicata since he is a fraud.

I can share you more evidence if anybody interested.

https://www.linkedin.com/feed/update/urn:li:activity:6906662303572606976/

![image](https://user-images.githubusercontent.com/72995735/157096643-3d6dd048-5dcf-47ca-b5c4-f97d4064134b.png)

**Comment by nasreenabdul** [comment]
I could feel that. 

You have done good job to let others know. I think We should complain to PPH too.

 

Regards,

**Comment by windrunnerdev** [comment]
@nasreenabdul , is he trying to scam you too?

**Comment by windrunnerdev** [comment]
I think we should open cloud forum to damage this fake FamilyTree365 business.

**Comment by shahghasiadil** [comment]
He is lier and haven't paid me yet

**Comment by shahghasiadil** [comment]
@windrunnerdev
@nasreenabdul
He has scammed me too

**Comment by nasreenabdul** [comment]
He tried but I got saved before I started work 😊 as I got another project and got busy in that.

 

You have done good job  to let everyone know. He is taking people from PPH. We should let PPH know so they disable his profile

**Comment by feci77** [comment]
Yes, he is taking people from PPH with higher rates and then try to scam
them.

I've worked with him for 3 months and after that, even paying me, He tried
to get money from me instead of paying me.
